asterisk/rootfs/etc/services.d/discovery/run

```diff
@@ -3,22 +3,27 @@
 # Home Assistant Community Add-on: Asterisk
 # Send Discovery information to Home Assistant
 # ==============================================================================
-declare config
 
-# Wait for motionEye to start before continuing
-bashio::net.wait_for 8089
+# shellcheck shell=bash
+
+# Wait for Asterisk to become available before continuing
+bashio::net.wait_for 8088
```

Comment:
I think it's more reliable to wait on port 8088, because there would not be any certificate issue.

Reply:
Good idea.